cmd/oras/internal/option/remote.go

```diff
@@ -116,19 +125,42 @@ func (opts *Remote) ApplyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description
 	fs.StringArrayVarP(&opts.headerFlags, flagPrefix+"header", shortHeader, nil, "add custom headers to "+notePrefix+"requests")
 }
 
+// CheckStdinConflict checks if opts.PasswordFromStdin or opts.IdentityTokenFromStdin
+// conflicts with read file from input.
+func (opts *Remote) CheckStdinConflict(passwordFromStdin bool, identityTokenFromStdin bool) error {
+	if passwordFromStdin {
+		return fmt.Errorf("`-` read file from input and `%s` read password from input cannot be both used", passwordFromStdinFlag)
+	} else if identityTokenFromStdin {
+		return fmt.Errorf("`-` read file from input and `%s` read identity token from input cannot be both used", identityTokenFromStdinFlag)
+	}
+	return nil
+}
```

Comment:
```suggestion
func (opts *Remote) CheckStdinConflict() error {
	if opts.PasswordFromStdin {
		return fmt.Errorf("`-` read file from input and `%s` read password from input cannot be both used", passwordFromStdinFlag)
	} else if opts.IdentityTokenFromStdin {
		return fmt.Errorf("`-` read file from input and `%s` read identity token from input cannot be both used", identityTokenFromStdinFlag)
	}
	return nil
}
```

Also missing e2e tests for the confliction check, can be added to command suite for `manifest` and `blob` commands

Reply:
added e2e tests